core/src/apps/management/reset_device.py

```diff
@@ -24,8 +24,10 @@
     # validate parameters and device state
     _validate_reset_device(msg)
 
+    is_slip39 = msg.backup_type == ResetDeviceType.Slip39_Single_Group
```

Comment:
this should include multi-group, no?

Reply:
The multigroup is not yet implemented and `_validate_reset_device` raises an error if such value is received, so it doesn't matter. If it seems more consistent, I can add it

Reply:
either that or call the variable `is_slip39_simple`